test/parallel/test-process-execve-no-args.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { skip, isWindows, isIBMi } = require('../common');
+const { fail } = require('assert');
+const { isMainThread } = require('worker_threads');
+const { spawnSync } = require('child_process');
+
+if (!isMainThread) {
+  skip('process.execve is not available in Workers');
+} else if (isWindows || isIBMi) {
+  skip('process.execve is not available in Windows or IBM i');
+}
+
+// Get full path to the ps executable
```

Comment:
This comment no longer matches the code.

Reply:
@richardlau fixed. I'm not sure what Node's etiquette for resolving PR comments, so to be on the safe side I'll leave the comment unresolved so you can resolve it if you think the comment is OK now.